doc/stdenv/meta.chapter.md

```diff
@@ -134,7 +134,33 @@ Attribute Set `lib.platforms` defines [various common lists](https://github.com/
 This attribute is special in that it is not actually under the `meta` attribute set but rather under the `passthru` attribute set. This is due to how `meta` attributes work, and the fact that they are supposed to contain only metadata, not derivations.
 :::
 
-An attribute set with as values tests. A test is a derivation, which builds successfully when the test passes, and fails to build otherwise. A derivation that is a test needs to have `meta.timeout` defined.
+An attribute set with tests as values. A test is a derivation that builds when the test passes and fails to build otherwise.
+
+#### Regular tests
+
+Tests that are part of the source package are often executed in the `installCheckPhase`.
+
+Prefer `passthru.tests` for tests that are introduced in nixpkgs because:
+
+* `passthru.tests` tests the 'real' package, independently from the environment in which it was built
+* we can run `passthru.tests` independently
+* `installCheckPhase` adds overhead to each build
+
+```nix
+{ /* ... */, key }:
+
+mkDerivation rec {
+  # ...
```

Comment:
```suggestion
  ...
```

Reply:
I disagree. `# ...` is clearly intended as meta-syntax, whereas `...` is close to actual syntax. It's a valid token in the language.

Reply:
@SuperSandro2000 can you clarify why you wanted `# ...` to be replaced by `...`, so we can perhaps find a third alternative that is better than the 2 competing ones here?

Reply:
(I moved this example to doc/contributing/coding-conventions.chapter.md and changed this to `…` (UTF-8 ellipsis) to be consistent with the other already-existing example there)